conformance/conformance_test.cc

```diff
@@ -1771,6 +1771,10 @@ bool ConformanceTestSuite::RunSuite(ConformanceTestRunner* runner,
       R"({"repeatedDuration": ["1.5s", "-1.5s"]})",
       "repeated_duration: {seconds: 1 nanos: 500000000}"
       "repeated_duration: {seconds: -1 nanos: -500000000}");
+  RunValidJsonTest(
+      "DurationNull", REQUIRED,
+      R"({"optionalDuration": null})",
+      "optional_duration: {seconds: 0}");
```

Comment:
{"optionalDuration": null} is equivalent to {} so the expected result is just an empty object, i.e., the test case should be:
```cpp
RunValidJsonTest(
    "DurationNull", REQUIRED,
    R"({"optionalDuration": null})",
    "");
```